api/api.py

```diff
@@ -16,195 +22,158 @@ def json_serial(obj):
 
 
 #ESTABLISHING CONNECTION WITH THE POSTGRES DATABASE
-cur= Session.connect_to_psql()
+ss=Session()
+session=ss.connect_to_psql()
+
+
+Blocks=ss.get_table_object('blocks')
```

Comment:
I am not sure if we should do this, but is it possible to connect the `blocks` tabls in the psql to the `ether_sql.models.Blocks` class in `ether_sql` library? 

If we can attach the two then we can reuse the functions defined in those classes.

Reply:
Check my reply on that issue you've created regarding this. I am fine with either approaches.